app/services/hyrax/file_set_characterizer.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Hyrax
+  ##
+  # Characterizes an ActiveFedora based FileSet
+  class FileSetCharacterizer
+    ##
+    # @!attribute [rw] source
+    #   @return [#characterize]
```

Comment:
I'm a bit fuzzy on this. Is this saying that the thing in the `source` attr responds to `#characterize`. Does it? I would expect `source` to return an instance of an AF-based  file set.